packages/jwt-verifier/package.json

```diff
@@ -7,14 +7,16 @@
   "main": "lib.js",
   "config": {
     "tck": {
-      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.1-SNAPSHOT&e=jar&c=shaded",
-      "setup_env": "cross-env ISSUER=http://localhost:9090/oauth2/default cross-env CLIENT_ID=OOICU812 cross-env CLIENT_SECRET=VERY_SECRET",
-      "command": "java -Dconfig=./test/integration-test/resources/testRunner.yml -cp ./okta-oidc-tck-0.2.1-SNAPSHOT-shaded.jar org.testng.TestNG -d target/cli-test-output ./test/integration-test/resources/testng.xml"
+      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.2-SNAPSHOT&e=jar&c=shaded",
+      "setup_env": "export ISSUER=http://localhost:9090/oauth2/default export CLIENT_ID=OOICU812 export CLIENT_SECRET=VERY_SECRET",
```

Comment:
Can't use `cross-env` as we start another process when TCK is run, and `cross-env` doesn't set env vars across processes.